src/client/utils/hooks/useQuoteCartData.ts

```diff
@@ -120,18 +124,19 @@ const getHouseDetails = (data: GenericQuoteData) => {
       },
     })
   } else if (data.yearOfOwnership) {
-    houseDetails.push(
-      {
-        label: 'CHECKOUT_DETAILS_YEAR_OF_OWNERSHIP',
-        value: { value: data.yearOfOwnership },
-      },
-      {
-        label: 'CHECKOUT_DETAILS_WATER_LEAKAGE',
-        value: {
-          textKey: data.waterLeakageDetector ? 'YES' : 'NO',
-        },
+    houseDetails.push({
+      label: 'CHECKOUT_DETAILS_YEAR_OF_OWNERSHIP',
+      value: { value: data.yearOfOwnership },
+    })
+  }
+
+  if (data.type === InsuranceType.NORWEGIAN_HOUSE) {
+    houseDetails.push({
+      label: 'CHECKOUT_DETAILS_WATER_LEAKAGE',
+      value: {
+        textKey: data.waterLeakageDetector ? 'YES' : 'NO',
```

Comment:
Anyway we can check if the value  is null instead. I assume, in NO, the value will always be set ("true" or "false), but it should be null or undefined if it does not exist. That should allow us to avoid the if statement.

Reply:
Thought the same thing but the issue is that is defined like this `waterLeakageDetector?: boolean`. When it's `null` it means 'Not Sure'.

Reply:
This should be modelled as an enum :) => yes, no, not_sure

Reply:
That's right